src/replication/CreateReplicationRequest.ts

```diff
@@ -0,0 +1,12 @@
+export default interface CreateReplicationRequest {
+    datasetId: string,
+    replica: number,
+    criteria: string,
+    client: string,
+    urlPrefix: string,
+    maxPrice: number,
+    maxNumberOfDeals: number,
+    isVerfied: string,
```

Comment:
why not use a boolean (isVerified and isOffline)

Reply:
Because it comes from cli arguments - I thought only string is supported.